content/release-10-10-0/overview-10-10-0.md

```diff
@@ -7,6 +7,8 @@ layout: bundle
 
 These release notes for **Cumulocity IoT release 10.10.0**, include all relevant changes (migration notes, improvements and fixes) that have been implemented in Cumulocity IoT Core, Cumulocity IoT Streaming Analytics, Cumulocity IoT Machine Learning, and Cumulocity IoT DataHub since the previous GA release.
 
+You can find the corresponding product documentation for release 10.10.0 at [cumulocity.com/guides/10.10.0/welcome/intro-documentation](https://cumulocity.com/guides/10.10.0/welcome/intro-documentation/).
```

Comment:
I would actually add https:// to the link her and in all other cases.

Reply:
Done